<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Farbe</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSlider" name="slider_max">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QSlider" name="slider_min">
          <property name="minimum">
           <number>-250</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>-50</number>
          </property>
          <property name="sliderPosition">
           <number>-50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cb_farbe"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Wertebereich Maximum</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Darstellung Plot</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Wertebereich Minimum</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="cb_darstellung"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="le_fkt"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Funktion Eingeben</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Anzahl Punkte</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="pb_plot">
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QSlider" name="slider_anzahl_punkte">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>110</number>
          </property>
          <property name="sliderPosition">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
